zetaclient/broadcast.go

```diff
@@ -41,6 +41,7 @@ func (b *ZetaCoreBridge) Broadcast(gaslimit uint64, authzWrappedMsg sdktypes.Msg
 	if baseGasPrice == 0 {
 		baseGasPrice = DefaultBaseGasPrice // shoudn't happen, but just in case
 	}
+	baseGasPrice = int64(float64(baseGasPrice) * 0.011)
```

Comment:
Why do we do this?
Need comments

Reply:
We should declare the constants in the common package and used to calculate the fee both here and in the antehandle . `feetx.GetGas()` provides access to the gas limit provided to a tx

Reply:
factored out the contant and use it in both zetacored and zetaclientd